index.html

```diff
@@ -21,5 +21,6 @@
       // Place the H1 element inside the div
       app.appendChild(header);
     </script>
+    <h2><a href="https://www.khanacademy.org/computer-programming/test-for-conrads-creative-mode-20/6404159454363648">Test for "Conrad's Creative Mode 2.0"</a></h2><script src="https://www.khanacademy.org/computer-programming/test-for-conrads-creative-mode-20/6404159454363648/embed.js?editor=yes&buttons=yes&author=yes&embed=yes"></script><p>Made using: <a href="http://www.khanacademy.org/computer-programming">Khan Academy Computer Science</a></p>
```

Comment:
Note: I expect this simple embed to work. If it does not, I may have to table until further notice.